packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/TokenScopeHandler.ts

```diff
@@ -42,12 +42,28 @@ export class TokenScopeHandler extends NestedScopeHandler {
     } = scopeA;
     const { identifierMatcher } = getMatcher(document.languageId);
 
+    const textA = document.getText(scopeA.domain);
+
     // NB: Don't directly use `test` here because global regexes are stateful
     // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#finding_successive_matches
-    return testRegex(identifierMatcher, document.getText(scopeA.domain))
-      ? true
-      : testRegex(identifierMatcher, document.getText(scopeB.domain))
-      ? false
-      : undefined;
+    if (testRegex(identifierMatcher, textA)) {
```

Comment:
Why not just add $ to identifierMatcher regex?

Reply:
Because that is a larger change. We discussed this in the linked issue.

Reply:
No I don't mean change the global identifier regex, just lump it in here. I'm not sure we should be preferring identifiers over $, as you're doing here, and it's simpler if we don't

Reply:
I do think we should prefer alpha to dollar